docker-compose.yml

```diff
@@ -0,0 +1,58 @@
+version: '3'
```

Comment:
nit: `version` in compose files is only informative and might be skipped (https://docs.docker.com/compose/compose-file/04-version-and-name/)

Reply:
Fine, removed.